src/scripts/quests/QuestLineHelper.ts

```diff
@@ -258,6 +258,37 @@ class QuestLineHelper {
         App.game.quests.questLines().push(RocketjohtoQuestLine);
     }
 
+    public static createAquaMagmahoennQuestLine() {
+        const AquaMagmahoennQuestLine = new QuestLine('Land vs Water', 'Put a stop to the schemes of Team Aqua and Team Magma');
+
+        const clearMtChimney = new CustomQuest(1, 0, 'Stop Team Magma at Mt. Chimney', () => App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex('Mt. Chimney')](), 0);
```

Comment:
Just checking that you expect these quest to auto complete if someone has already defeated all these dungeons?

Reply:
Yes that is what I expect.